lms/djangoapps/teams/static/teams/js/views/create_team.js

```diff
@@ -0,0 +1,43 @@
+;(function (define) {
+'use strict';
+
+define(['backbone',
+        'underscore',
+        'gettext',
+        'js/components/header/models/header',
+        'js/components/header/views/header',
+        'teams/js/views/edit_team'],
+       function (Backbone, _, gettext, HeaderModel, HeaderView, TeamEditView) {
+           return Backbone.View.extend({
```

Comment:
Instead of declaring a new Backbone view, could you use `ViewWithHeader` from `teams_tab.js`? That view's render method uses `setElement` which works a lot better than `jQuery.append`